target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -215,6 +215,14 @@ kng_rc)
 	set_usb_led "$board:green:usb"
 	set_wifi_led "$board:green:wifi"
 	;;
+maestro-e220)
+	set_wifi_led "$board:blue:wifi"
+	ucidef_set_led_default "activity" "activity" "$board:amber:activity" "1"
```

Comment:
Switching all LEDs on doesn't make any sense to me. Could it be that polarity of the GPIOs is wrong (active high instead of active low?)